### ISSUE TITLE
what's it happen?

### ISSUE THREAD

**Issue by lmx1989219** [spring-projects/spring-loaded#198]
when i recompile class ,wrong message occur

follow this:
2017-04-27 18:34:46.438 ERROR 2516 --- [Loader@18b4aac2] org.springsource.loaded.ReloadableType   : Unable to find regeneration methods for cglib proxies - proxies will be out of date for this type

**Comment by suiwenfeng** [comment]
```
2017-07-04 14:25:46.871 ERROR 82936 --- [Loader@18b4aac2] org.springsource.loaded.ReloadableType   : Unable to find regeneration methods for cglib proxies - proxies will be out of date for this type
2017-07-04 14:25:46.909 ERROR 82936 --- [Loader@18b4aac2] org.springsource.loaded.ReloadableType   : Unable to find regeneration methods for cglib proxies - proxies will be out of date for this type
2017-07-04 14:25:46.916 ERROR 82936 --- [Loader@18b4aac2] org.springsource.loaded.ReloadableType   : Unable to find regeneration methods for cglib proxies - proxies will be out of date for this type
2017-07-04 14:25:46.922 ERROR 82936 --- [Loader@18b4aac2] org.springsource.loaded.ReloadableType   : Unable to find regeneration methods for cglib proxies - proxies will be out of date for this type
2017-07-04 14:25:46.938 ERROR 82936 --- [Loader@18b4aac2] org.springsource.loaded.ReloadableType   : Unable to find regeneration methods for cglib proxies - proxies will be out of date for this type
2017-07-04 14:25:46.990 ERROR 82936 --- [Loader@18b4aac2] org.springsource.loaded.ReloadableType   : Unable to find regeneration methods for cglib proxies - proxies will be out of date for this type
2017-07-04 14:25:47.029 ERROR 82936 --- [Loader@18b4aac2] org.springsource.loaded.ReloadableType   : Unable to find regeneration methods for cglib proxies - proxies will be out of date for this type
2017-07-04 14:25:47.069 ERROR 82936 --- [Loader@18b4aac2] org.springsource.loaded.ReloadableType   : Unable to find regeneration methods for cglib proxies - proxies will be out of date for this type
2017-07-04 14:25:47.072 ERROR 82936 --- [Loader@18b4aac2] org.springsource.loaded.ReloadableType   : Unable to find regeneration methods for cglib proxies - proxies will be out of date for this type
2017-07-04 14:25:47.120 ERROR 82936 --- [Loader@18b4aac2] org.springsource.loaded.ReloadableType   : Unable to find regeneration methods for cglib proxies - proxies will be out of date for this type
2017-07-04 14:25:47.138 ERROR 82936 --- [Loader@18b4aac2] org.springsource.loaded.ReloadableType   : Unable to find regeneration methods for cglib proxies - proxies will be out of date for this type
2017-07-04 14:25:47.215 ERROR 82936 --- [Loader@18b4aac2] org.springsource.loaded.ReloadableType   : Unable to find regeneration methods for cglib proxies - proxies will be out of date for this type
2017-07-04 14:25:48.329 ERROR 82936 --- [Loader@18b4aac2] org.springsource.loaded.ReloadableType   : Unable to find regeneration methods for cglib proxies - proxies will be out of date for this type
2017-07-04 14:25:48.334 ERROR 82936 --- [Loader@18b4aac2] org.springsource.loaded.ReloadableType   : Unable to find regeneration methods for cglib proxies - proxies will be out of date for this type
```

same issue, and followed by NoSuchMethodError
java.lang.NoSuchMethodError: UserRepository.findOne(Ljava/io/Serializable;)Ljava/lang/Object;

```
				<dependencies>
					<dependency>
						<groupId>org.springframework</groupId>
						<artifactId>springloaded</artifactId>
						<version>1.2.7.RELEASE</version>
					</dependency>
				</dependencies>
```

**Comment by quangthe** [comment]
Have same issue, there are a lot of ERROR in logs 

```
2017-11-21 00:18:14.265 ERROR 8756 --- [Loader@18b4aac2] org.springsource.loaded.ReloadableType   : Unable to find regeneration methods for cglib proxies - proxies will be out of date for this type
```

Spring boot 1.5.3
SpringLoaded 1.2.7

**Comment by aclement** [comment]
cglib messages are fixed now in 1.2.9 snapshots.
